spec/support/e2e_setup.rb

```diff
@@ -38,6 +38,13 @@ def mui_choose(choice, from:)
     end
   end
 
+  def mui_radio_choose(choice, from:)
```

Comment:
This replaces mui_choose? If so maybe delete the old function or comment it out?